mkdocs/requirements.txt

```diff
@@ -1,9 +1,10 @@
+markdown>=3.2
+mike>=1.0.1
+mkdocs==1.1
```

Comment:
mkdocs>=1.2.2

Reply:
Done

Reply:
Hmm... it still seems to say `mkdocs==1.1`? Could you please make this `mkdocs>=1.2.2`

Reply:
@sriumcp Sorry about that... I accidentally made the change in a different branch.